api/src/main/kotlin/handler/TimetableLectureReminderHandler.kt

```diff
@@ -0,0 +1,53 @@
+package com.wafflestudio.snutt.handler
+
+import com.wafflestudio.snutt.middleware.SnuttRestApiDefaultMiddleware
+import com.wafflestudio.snutt.timetablelecturereminder.dto.TimetableLectureReminderDto
+import com.wafflestudio.snutt.timetablelecturereminder.dto.request.TimetableLectureReminderModifyRequestDto
+import com.wafflestudio.snutt.timetablelecturereminder.dto.response.TimetableLectureRemindersWithTimetableIdResponse
+import com.wafflestudio.snutt.timetablelecturereminder.service.TimetableLectureReminderService
+import org.springframework.stereotype.Component
+import org.springframework.web.reactive.function.server.ServerRequest
+import org.springframework.web.reactive.function.server.ServerResponse
+import org.springframework.web.reactive.function.server.awaitBody
+
+@Component
+class TimetableLectureReminderHandler(
+    private val timetableLectureReminderService: TimetableLectureReminderService,
+    snuttRestApiDefaultMiddleware: SnuttRestApiDefaultMiddleware,
+) : ServiceHandler(snuttRestApiDefaultMiddleware) {
+    suspend fun getReminder(req: ServerRequest): ServerResponse =
+        handle(req) {
+            val timetableId = req.pathVariable("timetableId")
+            val timetableLectureId = req.pathVariable("timetableLectureId")
+            timetableLectureReminderService.getReminder(timetableId, timetableLectureId)?.let(::TimetableLectureReminderDto) ?: Unit
+        }
+
+    suspend fun getRemindersInActiveSemesterPrimaryTimetable(req: ServerRequest): ServerResponse =
+        handle(req) {
+            val userId = req.userId
+            timetableLectureReminderService.getRemindersInActiveSemesterPrimaryTimetable(userId).let {
+                TimetableLectureRemindersWithTimetableIdResponse(
+                    timetableId = it.timetable.id,
+                    reminders = it.reminders.map(::TimetableLectureReminderDto),
+                )
+            }
```

Comment:
```suggestion
            val reminders = timetableLectureReminderService.getRemindersInActiveSemesterPrimaryTimetable(userId)
            TimetableLectureRemindersWithTimetableIdResponse(
                timetableId = it.timetable.id,
                reminders = it.reminders.map(::TimetableLectureReminderDto),
            )
```

별건아니지만 let 남용하면 가독성 떨어지는듯

Reply:
[ee60e6a](https://github.com/wafflestudio/snutt/pull/379/commits/ee60e6af26d849fd1eacc84d4018532f79469969)